### ISSUE TITLE
Unable to open the password tomb

### ISSUE THREAD

**Issue by Tiv0w** [roddhjav/pass-tomb#29]
So recently I ran into the same issue multiple times.
I want to open my password tomb as usual, so my operations are:
- I run `sudo swapoff -a` to disable swap before opening the pass-tomb
- it seems my swap is correctly emptied and disabled
- I run `pass open`, decrypt my key with no issue
and I see a 

> `[x] Error: Unable to open the password tomb.`

Here is the interesting part when running with the --debug flag:
```
.  Cannot initialize device-mapper. Is dm_mod kernel module loaded?
.  Cannot use device tomb..password.1588584847.loop12, name is invalid or still in use.
.  tomb [E] Failure mounting the encrypted file.
```
But when I reboot, there's a 1/2 chance everything's working.
I'm willing to provide info for debugging purposes.
Thanks in advance to anyone taking the time to respond :slightly_smiling_face:

**Comment by roddhjav** [comment]
According to the debug message, `dm_mod` is not loaded; therefore, you should load it with `sudo modprobe dm_mod`. And if it works load it on boot with something like: `echo 'options dm_mod' > /etc/modprobe.d/dmcrypt.conf`

**Comment by Tiv0w** [comment]
I needed the password tomb quickly, so rebooted. I'll try it after work and give you feedback

**Comment by Tiv0w** [comment]
Ok so I narrowed down the issue. I'm on Manjaro fresh, so I get lots of kernel updates. It seems like `dm_mod` is not loaded on boot, and I do my upgrades as usual. Then I try to run `pass-tomb` (which indirectly needs dm-crypt if I'm correct), but `modprobe` fails to load the `dm_mod` at the old kernel modules location (which has been replaced). :facepalm: 

I must admit, I'm a newbie with kernel and modules :upside_down_face:.

So I'll try to "eagerly" load `dm_mod` on boot, as you said. I'll let you know how it went and close the issue if it works.
